subprojects/docs/src/docs/release/notes.md

```diff
@@ -74,6 +74,16 @@ main(MavenPublication) {
 
 See the documentation for more information on [Gradle Module Metadata generation](userguide/publishing_gradle_module_metadata.html#sub:gmm-reproducible).
 
+## Variant-aware dependency substitution rules
+
+Gradle now supports substitution of dependencies with classifiers, attributes or capabilities.
```

Comment:
I would move that sentence below the part that explains the previous limitations.